app/src/main/java/io/gnosis/safe/ui/transaction/TransactionsViewModel.kt

```diff
@@ -19,7 +19,8 @@ class TransactionsViewModel
     fun load() {
         safeLaunch {
             val safeAddress = safeRepository.getActiveSafe()!!.address
```

Comment:
this will crash if we remove last safe on the device. Active safe will be null. We should check for that and load only if we have active safe

Reply:
This can be addressed in a different PR, I would say :) since I am not really working with transaction loading here.

Reply:
sure, but let's have an issue open

Reply:
https://github.com/gnosis/safe-android/issues/684